tutorials/nemotron_340B_synthetic_datagen/synthetic_preference_data_generation_nemotron_4_340B.ipynb

```diff
@@ -208,19 +208,17 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "async def generate_subtopics(client, topic, n_subtopics):\n",
+    "async def generate_subtopics(client, topic, n_subtopics):  # noqa: ANN001, ANN201\n",
```

Comment:
We should be able to grab these type hints from elsewhere in the notebook I think.